src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -116,13 +116,15 @@ struct patcher
     unknown_symbol_kind = 8
   };
 
-   enum class buf_type {
+  enum buf_type {
```

Comment:
We have to use enum, not "enum class" because I declared an array: Section_Name_Array[patcher::buf_type::buf_type_count] 

enum class does not work for me